example/README.md

```diff
@@ -239,14 +239,16 @@ A new folder should be added in the `/spec` folder for the workflow and the test
 
 Ideally the test can run in parallel with other tests and should be put in the `parallel` folder. If it cannot be, it should go in the `spec` folder. Only if the test should be run outside of the test suite should it go in the `standalone` folder.
 
-## Fake data server
+## Fake Data Server
 
 A fake server is required for tests testing FTP/HTTP/HTTPS discover and downloads. The fake server should be set up once per account.
 
-The Cloudformation template for the fake data server is in `fake-server.yml`. To setup the fake server run:
+The Cloudformation template for the fake data server is in `fake-provider-cf.yml`. To setup the fake server, you can use the AWS CLI or AWS Console.
+
+If you want to use the AWS CLI, run:
 
 ```bash
-aws cloudformation deploy --template-file fake-server.yml --stack-name <stack-name> --parameter-overrides VpcId=<vpc-XXXXX> SubnetId=<subnet-XXXXXX> AZone=<az-zone> Ngap=true --capabilities CAPABILITY_NAMED_IAM
+aws cloudformation deploy --template-file fake-provider-cf.yml --stack-name <stack-name> --parameter-overrides VpcId=<vpc-XXXXX> SubnetId=<subnet-XXXXXX> AZone=<az-zone> Ngap=true --capabilities CAPABILITY_NAMED_IAM
```

Comment:
I don't think this template supports an `AZone` parameter or `Ngap` parameters. And `SubnetId` is actually `Subnet` for this template. Plus, there are additional parameters for `Bucket`, `FtpPassword`, `NGAPProtAppInstanceMinimalPolicyName`, `LatestAmiId`, and `PermissionsBoundary`. I know that you had to remove some of them to deploy your template outside of NGAP.

If you want to at least remove AZone and Ngap and rename SubnetId -> Subnet, I can take care of adding the documentation for the other parameters.

Reply:
I've updated the parameters list but am not sure what description to put for Bucket, FtpPassword, NGAPProtAppInstanceMinimalPolicyName. If you could look over all the descriptions and suggest some for `FtpPassword` and `Bucket`, that would be great!